mk/config.mk

```diff
@@ -227,3 +227,7 @@ endif
 # CFG_GP_SOCKETS
 # Enable Global Platform Sockets support
 CFG_GP_SOCKETS ?= y
+
+# Enable Secure Data Path support in OP-TEE core (TA may be invoked with
+# invocation parameters referring to specific secure memories).
```

Comment:
Minor: `s/in OP-TEE core//`.
Also, since this commit only introduces the basic infrastructure for SDP, but nothing can be done with it yet, I think it would be better to not change `config.mk` in this commit. Instead, insert this section in the last commit, when everything is there ready to be used. Config variables default to `n` (disabled), anyway.